programmers-coding-test/kimjiwon/implement/시각.js

```diff
@@ -0,0 +1,16 @@
+const rl = require("readline-sync");
+
+const H = +rl.question("몇 시까지 할껀데 딱 말혀");
+let count = 0;
+
+for (let h = 0; h < H + 1; h++) {
+  for (let m = 0; m < 60; m++) {
+    for (let s = 0; s < 60; s++) {
+      let time = `${h}:${m}:${s}`;
+      if (time.match(/[3]/g)) {
```

Comment:
오 신박하긴한데, 정규표현식은 안쓸 수 있다면 안쓰는게 좋은 것 같음! 가독성 때문.
```suggestion
      if (time.includes(“3”) {
```